common/log/loggerimpl/logger.go

```diff
@@ -18,50 +18,48 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package log
+package loggerimpl
 
 import (
 	"fmt"
-	"path/filepath"
-	"runtime"
-
+	"github.com/stretchr/testify/suite"
```

Comment:
import order

Reply:
Yes, I will clean up all the import order before checking in